packages/wdio-runner/src/index.ts

```diff
@@ -5,19 +5,49 @@ import EventEmitter from 'events'
 
 import logger from '@wdio/logger'
 import { initialiseWorkerService, initialisePlugin, executeHooksWithArgs } from '@wdio/utils'
-import { ConfigParser } from '@wdio/config'
+import { ConfigParser, ConfigOptions, SingleConfigOption, Capability } from '@wdio/config'
 
 import BaseReporter from './reporter'
 import { runHook, initialiseInstance, filterLogTypes, getInstancesData } from './utils'
 
 const log = logger('@wdio/runner')
 
+interface Args extends Partial<ConfigOptions> {
+    ignoredWorkerServices?: string[]
+}
+
+type RunParams = {
+    cid: string
+    args: Args
+    specs: string[]
+    caps: Capability
+    configFile: string
+    retries: number
+}
+
+interface Framework {
```

Comment:
maybe it can be renamed to test framework or similar because Framework looks too generic